src/rtos/rtos.c

```diff
@@ -789,10 +790,29 @@ static int rtos_try_next(struct target *target)
 	return 1;
 }
 
-int rtos_update_threads(struct target *target)
+struct rtos *rtos_of_target(struct target *target)
 {
+	/* Primarily consider the rtos field of the target itself, secondarily consider
+		rtos field SMP leader target, then consider rtos field of any other target in the SMP group.
+		Otherwise NULL return means that no associated non-zero rtos field could be found. */
```

Comment:
Something is wonky with the indenting here.

Reply:
Latest revision fixes the indentation.  Looks like my commit had the side-effect of github invalidating your prior approval, so I guess this needs an approval again, before I'm allowed to press the merge button.